editions/tw5.com/tiddlers/$__config_ViewToolbarButtons_Visibility_$__core_ui_Buttons_info.tid

```diff
@@ -0,0 +1,6 @@
+created: 20220413102320003
+modified: 20220413102320003
+title: $:/config/ViewToolbarButtons/Visibility/$:/core/ui/Buttons/info
```

Comment:
Did you mean to add this config tiddler?

Reply:
No will remove it